{"log":"W0206 10:22:15.011075       1 server.go:185] WARNING: all flags other than --config, --write-config-to, and --cleanup are deprecated. Please begin using a config file ASAP.\n","stream":"stderr","time":"2018-02-06T10:22:15.011451908Z"}
{"log":"I0206 10:22:15.133693       1 server_others.go:138] Using iptables Proxier.\n","stream":"stderr","time":"2018-02-06T10:22:15.134151931Z"}
{"log":"I0206 10:22:15.150973       1 server_others.go:171] Tearing down inactive rules.\n","stream":"stderr","time":"2018-02-06T10:22:15.15118689Z"}
{"log":"I0206 10:22:15.194374       1 server.go:426] Version: v1.9.2\n","stream":"stderr","time":"2018-02-06T10:22:15.208178715Z"}
{"log":"I0206 10:22:15.200471       1 conntrack.go:98] Set sysctl 'net/netfilter/nf_conntrack_max' to 131072\n","stream":"stderr","time":"2018-02-06T10:22:15.208208432Z"}
{"log":"I0206 10:22:15.200665       1 conntrack.go:52] Setting nf_conntrack_max to 131072\n","stream":"stderr","time":"2018-02-06T10:22:15.208226382Z"}
{"log":"I0206 10:22:15.201086       1 conntrack.go:83] Setting conntrack hashsize to 32768\n","stream":"stderr","time":"2018-02-06T10:22:15.208231075Z"}
{"log":"I0206 10:22:15.206316       1 conntrack.go:98] Set sysctl 'net/netfilter/nf_conntrack_tcp_timeout_established' to 86400\n","stream":"stderr","time":"2018-02-06T10:22:15.209252487Z"}
{"log":"I0206 10:22:15.206462       1 conntrack.go:98] Set sysctl 'net/netfilter/nf_conntrack_tcp_timeout_close_wait' to 3600\n","stream":"stderr","time":"2018-02-06T10:22:15.209289648Z"}
{"log":"I0206 10:22:15.206670       1 config.go:102] Starting endpoints config controller\n","stream":"stderr","time":"2018-02-06T10:22:15.20929511Z"}
{"log":"I0206 10:22:15.206686       1 controller_utils.go:1019] Waiting for caches to sync for endpoints config controller\n","stream":"stderr","time":"2018-02-06T10:22:15.20929947Z"}
{"log":"I0206 10:22:15.206714       1 config.go:202] Starting service config controller\n","stream":"stderr","time":"2018-02-06T10:22:15.209303642Z"}
{"log":"I0206 10:22:15.206723       1 controller_utils.go:1019] Waiting for caches to sync for service config controller\n","stream":"stderr","time":"2018-02-06T10:22:15.209307532Z"}
{"log":"I0206 10:22:15.306885       1 controller_utils.go:1026] Caches are synced for service config controller\n","stream":"stderr","time":"2018-02-06T10:22:15.343438525Z"}
{"log":"I0206 10:22:15.306995       1 controller_utils.go:1026] Caches are synced for endpoints config controller\n","stream":"stderr","time":"2018-02-06T10:22:15.343466126Z"}
{"log":"E0206 10:22:49.329804       1 streamwatcher.go:109] Unable to decode an event from the watch stream: http2: server sent GOAWAY and closed the connection; LastStreamID=11, ErrCode=NO_ERROR, debug=\"\"\n","stream":"stderr","time":"2018-02-06T10:22:49.338423024Z"}
{"log":"E0206 10:22:49.335379       1 streamwatcher.go:109] Unable to decode an event from the watch stream: http2: server sent GOAWAY and closed the connection; LastStreamID=11, ErrCode=NO_ERROR, debug=\"\"\n","stream":"stderr","time":"2018-02-06T10:22:49.338472721Z"}
{"log":"E0206 10:22:49.354701       1 reflector.go:322] k8s.io/kubernetes/pkg/client/informers/informers_generated/internalversion/factory.go:85: Failed to watch *core.Service: Get https://k8s.test.logrange.io:443/api/v1/services?resourceVersion=243\u0026timeoutSeconds=506\u0026watch=true: dial tcp 159.89.147.46:443: getsockopt: connection refused\n","stream":"stderr","time":"2018-02-06T10:22:49.356662102Z"}
{"log":"E0206 10:22:49.354779       1 reflector.go:322] k8s.io/kubernetes/pkg/client/informers/informers_generated/internalversion/factory.go:85: Failed to watch *core.Endpoints: Get https://k8s.test.logrange.io:443/api/v1/endpoints?resourceVersion=635\u0026timeoutSeconds=427\u0026watch=true: dial tcp 159.89.147.46:443: getsockopt: connection refused\n","stream":"stderr","time":"2018-02-06T10:22:49.356701052Z"}
{"log":"E0206 10:22:50.356290       1 reflector.go:205] k8s.io/kubernetes/pkg/client/informers/informers_generated/internalversion/factory.go:85: Failed to list *core.Service: Get https://k8s.test.logrange.io:443/api/v1/services?limit=500\u0026resourceVersion=0: dial tcp 159.89.147.46:443: getsockopt: connection refused\n","stream":"stderr","time":"2018-02-06T10:22:50.44303119Z"}
{"log":"E0206 10:22:50.366769       1 reflector.go:205] k8s.io/kubernetes/pkg/client/informers/informers_generated/internalversion/factory.go:85: Failed to list *core.Endpoints: Get https://k8s.test.logrange.io:443/api/v1/endpoints?limit=500\u0026resourceVersion=0: dial tcp 159.89.147.46:443: getsockopt: connection refused\n","stream":"stderr","time":"2018-02-06T10:22:50.443055015Z"}
{"log":"E0206 10:22:51.358151       1 reflector.go:205] k8s.io/kubernetes/pkg/client/informers/informers_generated/internalversion/factory.go:85: Failed to list *core.Service: Get https://k8s.test.logrange.io:443/api/v1/services?limit=500\u0026resourceVersion=0: dial tcp 159.89.147.46:443: getsockopt: connection refused\n","stream":"stderr","time":"2018-02-06T10:22:51.358383179Z"}
{"log":"E0206 10:22:51.368402       1 reflector.go:205] k8s.io/kubernetes/pkg/client/informers/informers_generated/internalversion/factory.go:85: Failed to list *core.Endpoints: Get https://k8s.test.logrange.io:443/api/v1/endpoints?limit=500\u0026resourceVersion=0: dial tcp 159.89.147.46:443: getsockopt: connection refused\n","stream":"stderr","time":"2018-02-06T10:22:51.368575244Z"}
{"log":"E0206 10:22:52.359983       1 reflector.go:205] k8s.io/kubernetes/pkg/client/informers/informers_generated/internalversion/factory.go:85: Failed to list *core.Service: Get https://k8s.test.logrange.io:443/api/v1/services?limit=500\u0026resourceVersion=0: dial tcp 159.89.147.46:443: getsockopt: connection refused\n","stream":"stderr","time":"2018-02-06T10:22:52.360264095Z"}
{"log":"E0206 10:22:52.370104       1 reflector.go:205] k8s.io/kubernetes/pkg/client/informers/informers_generated/internalversion/factory.go:85: Failed to list *core.Endpoints: Get https://k8s.test.logrange.io:443/api/v1/endpoints?limit=500\u0026resourceVersion=0: dial tcp 159.89.147.46:443: getsockopt: connection refused\n","stream":"stderr","time":"2018-02-06T10:22:52.370300157Z"}
{"log":"E0206 10:22:53.361856       1 reflector.go:205] k8s.io/kubernetes/pkg/client/informers/informers_generated/internalversion/factory.go:85: Failed to list *core.Service: Get https://k8s.test.logrange.io:443/api/v1/services?limit=500\u0026resourceVersion=0: dial tcp 159.89.147.46:443: getsockopt: connection refused\n","stream":"stderr","time":"2018-02-06T10:22:53.362090439Z"}
{"log":"E0206 10:22:53.372409       1 reflector.go:205] k8s.io/kubernetes/pkg/client/informers/informers_generated/internalversion/factory.go:85: Failed to list *core.Endpoints: Get https://k8s.test.logrange.io:443/api/v1/endpoints?limit=500\u0026resourceVersion=0: dial tcp 159.89.147.46:443: getsockopt: connection refused\n","stream":"stderr","time":"2018-02-06T10:22:53.383269428Z"}
{"log":"E0206 10:22:54.363627       1 reflector.go:205] k8s.io/kubernetes/pkg/client/informers/informers_generated/internalversion/factory.go:85: Failed to list *core.Service: Get https://k8s.test.logrange.io:443/api/v1/services?limit=500\u0026resourceVersion=0: dial tcp 159.89.147.46:443: getsockopt: connection refused\n","stream":"stderr","time":"2018-02-06T10:22:54.363997794Z"}
{"log":"E0206 10:22:54.375759       1 reflector.go:205] k8s.io/kubernetes/pkg/client/informers/informers_generated/internalversion/factory.go:85: Failed to list *core.Endpoints: Get https://k8s.test.logrange.io:443/api/v1/endpoints?limit=500\u0026resourceVersion=0: dial tcp 159.89.147.46:443: getsockopt: connection refused\n","stream":"stderr","time":"2018-02-06T10:22:54.375933122Z"}
{"log":"E0206 10:22:55.365505       1 reflector.go:205] k8s.io/kubernetes/pkg/client/informers/informers_generated/internalversion/factory.go:85: Failed to list *core.Service: Get https://k8s.test.logrange.io:443/api/v1/services?limit=500\u0026resourceVersion=0: dial tcp 159.89.147.46:443: getsockopt: connection refused\n","stream":"stderr","time":"2018-02-06T10:22:55.371585871Z"}
{"log":"E0206 10:22:55.377735       1 reflector.go:205] k8s.io/kubernetes/pkg/client/informers/informers_generated/internalversion/factory.go:85: Failed to list *core.Endpoints: Get https://k8s.test.logrange.io:443/api/v1/endpoints?limit=500\u0026resourceVersion=0: dial tcp 159.89.147.46:443: getsockopt: connection refused\n","stream":"stderr","time":"2018-02-06T10:22:55.377973951Z"}
{"log":"E0206 10:22:56.376580       1 reflector.go:205] k8s.io/kubernetes/pkg/client/informers/informers_generated/internalversion/factory.go:85: Failed to list *core.Service: Get https://k8s.test.logrange.io:443/api/v1/services?limit=500\u0026resourceVersion=0: dial tcp 159.89.147.46:443: getsockopt: connection refused\n","stream":"stderr","time":"2018-02-06T10:22:56.37676667Z"}
{"log":"E0206 10:22:56.386549       1 reflector.go:205] k8s.io/kubernetes/pkg/client/informers/informers_generated/internalversion/factory.go:85: Failed to list *core.Endpoints: Get https://k8s.test.logrange.io:443/api/v1/endpoints?limit=500\u0026resourceVersion=0: dial tcp 159.89.147.46:443: getsockopt: connection refused\n","stream":"stderr","time":"2018-02-06T10:22:56.400468284Z"}
{"log":"E0206 10:22:57.378777       1 reflector.go:205] k8s.io/kubernetes/pkg/client/informers/informers_generated/internalversion/factory.go:85: Failed to list *core.Service: Get https://k8s.test.logrange.io:443/api/v1/services?limit=500\u0026resourceVersion=0: dial tcp 159.89.147.46:443: getsockopt: connection refused\n","stream":"stderr","time":"2018-02-06T10:22:57.379047078Z"}
{"log":"E0206 10:22:57.389561       1 reflector.go:205] k8s.io/kubernetes/pkg/client/informers/informers_generated/internalversion/factory.go:85: Failed to list *core.Endpoints: Get https://k8s.test.logrange.io:443/api/v1/endpoints?limit=500\u0026resourceVersion=0: dial tcp 159.89.147.46:443: getsockopt: connection refused\n","stream":"stderr","time":"2018-02-06T10:22:57.389664231Z"}
{"log":"E0206 10:22:58.380665       1 reflector.go:205] k8s.io/kubernetes/pkg/client/informers/informers_generated/internalversion/factory.go:85: Failed to list *core.Service: Get https://k8s.test.logrange.io:443/api/v1/services?limit=500\u0026resourceVersion=0: dial tcp 159.89.147.46:443: getsockopt: connection refused\n","stream":"stderr","time":"2018-02-06T10:22:58.380949768Z"}
{"log":"E0206 10:22:58.391655       1 reflector.go:205] k8s.io/kubernetes/pkg/client/informers/informers_generated/internalversion/factory.go:85: Failed to list *core.Endpoints: Get https://k8s.test.logrange.io:443/api/v1/endpoints?limit=500\u0026resourceVersion=0: dial tcp 159.89.147.46:443: getsockopt: connection refused\n","stream":"stderr","time":"2018-02-06T10:22:58.391806125Z"}
{"log":"E0206 10:22:59.382638       1 reflector.go:205] k8s.io/kubernetes/pkg/client/informers/informers_generated/internalversion/factory.go:85: Failed to list *core.Service: Get https://k8s.test.logrange.io:443/api/v1/services?limit=500\u0026resourceVersion=0: dial tcp 159.89.147.46:443: getsockopt: connection refused\n","stream":"stderr","time":"2018-02-06T10:22:59.3829394Z"}
{"log":"E0206 10:22:59.393193       1 reflector.go:205] k8s.io/kubernetes/pkg/client/informers/informers_generated/internalversion/factory.go:85: Failed to list *core.Endpoints: Get https://k8s.test.logrange.io:443/api/v1/endpoints?limit=500\u0026resourceVersion=0: dial tcp 159.89.147.46:443: getsockopt: connection refused\n","stream":"stderr","time":"2018-02-06T10:22:59.406440883Z"}
{"log":"E0206 10:23:00.397356       1 reflector.go:205] k8s.io/kubernetes/pkg/client/informers/informers_generated/internalversion/factory.go:85: Failed to list *core.Service: Get https://k8s.test.logrange.io:443/api/v1/services?limit=500\u0026resourceVersion=0: dial tcp 159.89.147.46:443: getsockopt: connection refused\n","stream":"stderr","time":"2018-02-06T10:23:00.397565152Z"}
{"log":"E0206 10:23:00.397469       1 reflector.go:205] k8s.io/kubernetes/pkg/client/informers/informers_generated/internalversion/factory.go:85: Failed to list *core.Endpoints: Get https://k8s.test.logrange.io:443/api/v1/endpoints?limit=500\u0026resourceVersion=0: dial tcp 159.89.147.46:443: getsockopt: connection refused\n","stream":"stderr","time":"2018-02-06T10:23:00.397589249Z"}
{"log":"E0206 10:23:01.398905       1 reflector.go:205] k8s.io/kubernetes/pkg/client/informers/informers_generated/internalversion/factory.go:85: Failed to list *core.Service: Get https://k8s.test.logrange.io:443/api/v1/services?limit=500\u0026resourceVersion=0: dial tcp 159.89.147.46:443: getsockopt: connection refused\n","stream":"stderr","time":"2018-02-06T10:23:01.399602812Z"}
{"log":"E0206 10:23:01.399891       1 reflector.go:205] k8s.io/kubernetes/pkg/client/informers/informers_generated/internalversion/factory.go:85: Failed to list *core.Endpoints: Get https://k8s.test.logrange.io:443/api/v1/endpoints?limit=500\u0026resourceVersion=0: dial tcp 159.89.147.46:443: getsockopt: connection refused\n","stream":"stderr","time":"2018-02-06T10:23:01.400027398Z"}
{"log":"E0206 10:23:08.191931       1 reflector.go:205] k8s.io/kubernetes/pkg/client/informers/informers_generated/internalversion/factory.go:85: Failed to list *core.Endpoints: endpoints is forbidden: User \"system:serviceaccount:kube-system:kube-proxy\" cannot list endpoints at the cluster scope\n","stream":"stderr","time":"2018-02-06T10:23:08.192149201Z"}
{"log":"E0206 10:23:08.192194       1 reflector.go:205] k8s.io/kubernetes/pkg/client/informers/informers_generated/internalversion/factory.go:85: Failed to list *core.Service: services is forbidden: User \"system:serviceaccount:kube-system:kube-proxy\" cannot list services at the cluster scope\n","stream":"stderr","time":"2018-02-06T10:23:08.192369871Z"}
